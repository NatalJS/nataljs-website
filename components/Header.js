import React from 'react';

const Header = () => (
  <header className="header">
    <div className="header__logo">
      <img className="logo--mobile" src="/static/images/logo-mobile.svg" alt="NatalJS Logo" />
      <img className="logo--desktop" src="/static/images/logo.svg" alt="NatalJS Logo" />
    </div>
    <menu className="header__menu">
      <ul className="menu">
        <li className="menu__item">
          <a href="#about" className="menu__item__link scroll">O que é</a>
        </li>
        <li className="menu__item">
          <a href="#how-to" className="menu__item__link scroll">Por onde começar?</a>
        </li>
        <li className="menu__item">
          <a href="#sign-up" className="menu__item__link scroll">Inscreva-se</a>
        </li>
        <li className="menu__item">
          <a href="#call-for-papers" className="menu__item__link scroll">Call for papers</a>
        </li>
        <li className="menu__item">
          <a href="#incoming-events" className="menu__item__link scroll">Próximos eventos</a>
        </li>
        <li className="menu__item">
          <a href="#join-us" className="menu__item__link scroll">Faça parte</a>
        </li>
      </ul>
    </menu>
  </header>
);

export default Header;
