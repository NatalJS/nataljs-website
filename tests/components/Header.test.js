import renderer from 'react-test-renderer';
import Header from 'components/Header';

describe('Component Header', () => {
  let Component;

  beforeAll(() => {
    Component = setup(Header);
  });

  it('should render component correctly', () => {
    const wrapper = renderer.create(Component());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
