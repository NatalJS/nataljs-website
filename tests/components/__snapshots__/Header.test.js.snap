// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header should render component correctly 1`] = `
<header
  className="header"
>
  <div
    className="header__logo"
  >
    <img
      alt="NatalJS Logo"
      className="logo--mobile"
      src="/static/images/logo-mobile.svg"
    />
    <img
      alt="NatalJS Logo"
      className="logo--desktop"
      src="/static/images/logo.svg"
    />
  </div>
  <menu
    className="header__menu"
  >
    <ul
      className="menu"
    >
      <li
        className="menu__item"
      >
        <a
          className="menu__item__link scroll"
          href="#about"
        >
          O que é
        </a>
      </li>
      <li
        className="menu__item"
      >
        <a
          className="menu__item__link scroll"
          href="#how-to"
        >
          Por onde começar?
        </a>
      </li>
      <li
        className="menu__item"
      >
        <a
          className="menu__item__link scroll"
          href="#sign-up"
        >
          Inscreva-se
        </a>
      </li>
      <li
        className="menu__item"
      >
        <a
          className="menu__item__link scroll"
          href="#call-for-papers"
        >
          Call for papers
        </a>
      </li>
      <li
        className="menu__item"
      >
        <a
          className="menu__item__link scroll"
          href="#incoming-events"
        >
          Próximos eventos
        </a>
      </li>
      <li
        className="menu__item"
      >
        <a
          className="menu__item__link scroll"
          href="#join-us"
        >
          Faça parte
        </a>
      </li>
    </ul>
  </menu>
</header>
`;
